README.md

```diff
@@ -1,2 +1,16 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+Display 10 random words    
+Number of attempts = 5     
+X = number of correct letter    
+    Output = choice of words   
+Input = Choose one word   
+While = number of attempts >0     
+	IF answer = "Password accepted"  
+   		Then output = next screen   
+	Else answer is wrong "Likeness = x"   
+   		Then input = choose a word   
+	Until Answer = "password accepted"    
```

Comment:
Control flow is unclear here -- is this an if statement or a loop? Best to stick to commonly understood control structures for clarity